internal/cmd/hana/map.go

```diff
@@ -80,6 +87,13 @@ func createHanaAPIInstance(config *hanaCheckConfig) error {
 }
 
 func createHanaAPIBinding(config *hanaCheckConfig) error {
+	//check if instance exists, skip API creation if it does
```

Comment:
Could we extract this code to another function? This check is not main responsibility of `createHanaAPIBinding`.